src/extension.ts

```diff
@@ -116,11 +122,18 @@ async function getThemeList(
 
 async function addCurrentTheme(extensionConfig = getExtensionConfig(), userSettings = getUserSettings()): Promise<void> {
   const themeList = await getThemeList(extensionConfig, userSettings);
-
   const currentThemeName: string | undefined = userSettings.get('workbench.colorTheme');
 
+  _addCurrentTheme(themeList, currentThemeName);
+}
+
+function _addCurrentTheme(themeList: string[], currentThemeName: string | string[] | undefined) {
   if (typeof currentThemeName !== 'undefined') {
-    saveThemes(themeList.concat(currentThemeName).sort(), Messages.AddedTheme(currentThemeName));
+    let keys = typeof currentThemeName === typeof Array ?
```

Comment:
Since here you are not modifying the variable in can be a `const`

Reply:
Good evening !
Thanks,
I work in a office so we don't use GitHub in this way;
How can i request you as a reviewer ?
Is there a button or something ?

Regarding the issue: Fixed !👍🏼
(i missed it while refactoring)

Reply:
here is the official documentation that can explain it better than me https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/requesting-a-pull-request-review